pkgs/tools/security/age-plugin-ledger/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, libusb1
+, openssl
+, rage
+, AppKit
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "age-plugin-ledger";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "Ledger-Donjon";
+    repo = pname;
```

```suggestion
    repo = "age-plugin-ledger";
```

we recently started using the repo name as string even if it's the same.